### PR TITLE
Link to add business added to the code

### PR DESCRIPTION
**Pull request by sergiobtos**

### CODE REVIEW

File: client/src/components/MyBusinessList/MyBusinessList.tsx [hunk 1]
```diff
@@ -45,32 +45,20 @@ export const MyBusinessList = () => {
         ) : (
           <>
             <div className='Profile-UserContainer_reviews business'>
-              <h4>your businesses</h4>
+              <h4>Your businesses</h4>
+              {/* temporary showing just business name because it was not showing anything */}
```
Comment (CodeMeKathy) [inline client/src/components/MyBusinessList/MyBusinessList.tsx:49]: Hi @sergiobtos!  May I suggest adding a `TODO` or `FIXME` tag at the beginning of your comment to highlight it better and assist in tracking the potential backlog functional and styling issues? 
 Thank you in advance for your consideration!

**Review by CodeMeKathy — CHANGES_REQUESTED** [review]
Thank you for your efforts and for taking the time. I recommend making a couple of changes after reviewing the comments and then resubmit the PR.

File: client/src/components/MyBusinessList/MyBusinessListOld.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ export const MyBusinessList = () => {
   const [list, setList] = useState<any[]>([])
   const [loading, setLoading] = useState(true)
   const history = useHistory()
-  const ownerId = JSON.parse(localStorage.getItem('profile') ?? 'false').result
+  const ownerId = JSON.parse(localStorage.getItem('profile') ?? 'false').data.result
```
[the remark this answers is not in the document]
Comment (sergiobtos) [reply client/src/components/MyBusinessList/MyBusinessListOld.tsx:12]: In my code does not work. i can show to you when you are available. If the data key word is not there my code crash.

File: client/src/components/MyBusinessList/MyBusinessList.tsx (hunk 1 above)
Comment (sergiobtos) [reply client/src/components/MyBusinessList/MyBusinessList.tsx]: Done

**Review by sergiobtos — COMMENTED** [review]
I did the change you requested about adding "TODO" in the code, thanks

**Review by CodeMeKathy — CHANGES_REQUESTED** [review]

File: client/package.json [hunk 3]
```diff
@@ -24,6 +24,7 @@
     "i": "^0.3.7",
     "jwt-decode": "^3.1.2",
     "npm": "^8.5.5",
+    "phosphor-react": "^1.4.1",
```
[the remark this answers is not in the document]
Comment (sergiobtos) [reply client/package.json:27]: I think not, let me delete here

**Pull request merged**